javascript/javascript2/week3/js2-week3-Homework/fastestPresser/fastestPresser.js

```diff
@@ -0,0 +1,71 @@
+"use strict"
+
+const timeInput = document.querySelector('#numberInput');
+const gameStart = document.querySelector('#btn1')
+const playerS = document.querySelector('.player-S')
+const playerL = document.querySelector('.player-L')
+const result = document.querySelector('.result')
+const restart = document.querySelector('.restart')
+const newUlTag = document.createElement('ul')
+const ulTag = document.createElement('ul')
+playerS.appendChild(newUlTag)
+playerL.appendChild(ulTag)
```

Comment:
put these two `ul` tags into the html, and then select them, like the other DOM elements. It's not good practice to mix static html DOM with static html DOM genereated by javascript. just choose the former.